CalculatorIdo/Style.css

```diff
@@ -0,0 +1,24 @@
+body {
+    display: flex;
+    justify-content: center;
+    height: 55em; /* Needed to use the align-items */
```

Comment:
It's not recommended to set the height of the body, it's better to create some sort of container.
Also, don't use em if the size isn't relative to the font size directly. I sent an article about correct CSS sizing.